Add wifi driver

francois-berder-imgtec opened the pull request.

nikhil-zinjurde-imgtec committed:
Merge pull request #1 from IMGCreator/dev

Merge request

nikhil-zinjurde-imgtec committed:
Merge pull request #21 from nikhil-zinjurde-imgtec/master

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #2 from IMGCreator/dev

Merge request

nikhil-zinjurde-imgtec committed:
Merge pull request #25 from nikhil-zinjurde-imgtec/master

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #3 from IMGCreator/dev

Merge request

nikhil-zinjurde-imgtec committed:
Merge pull request #28 from nikhil-zinjurde-imgtec/master

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #32 from Creatordev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #34 from Creatordev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #39 from Creatordev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #41 from Creatordev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #44 from Creatordev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
proddata feed: increase version to 0.1.1

Signed-off-by: Nikhil Zinjurde <nikhil.zinjurde@imgtec.com>

nikhil-zinjurde-imgtec committed:
Merge pull request #45 from nikhil-zinjurde-imgtec/update-proddata

proddata feed: increase version to 0.1.1

nikhil-zinjurde-imgtec committed:
Merge pull request #51 from Creatordev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #53 from Creatordev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #60 from CreatorDev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #62 from CreatorDev/dev

Merge dev to master

francois-berder committed:
Removing LetMeCreate library from openWrt-feeds

Moving LetMecreate to Creator-feed.

Signed-off-by: Francois Berder <Francois.Berder@imgtec.com>

nikhil-zinjurde-imgtec committed:
Merge pull request #65 from CreatorDev/dev

Merge dev to master

Commit:
Merge pull request #63 from francois-berder-imgtec/remove-letmecreate

Removing LetMeCreate library from openwrt-feeds

nikhil-zinjurde-imgtec committed:
Merge pull request #68 from CreatorDev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #72 from CreatorDev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #75 from CreatorDev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #76 from CreatorDev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #79 from CreatorDev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #80 from CreatorDev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #82 from CreatorDev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #86 from CreatorDev/dev

Merge dev to master

nikhil-zinjurde-imgtec committed:
Merge pull request #89 from CreatorDev/dev

Merge dev to master

francois-berder committed:
Add uccp420wlan feed

Signed-off-by: Francois Berder <francois.berder@imgtec.com>

nikhil-zinjurde-imgtec commented:
test with jenkins - dev

abhijit-mahajani-imgtec commented on uccp420wlan/files/etc/init.d/uccp420wlan:
```diff
@@ -0,0 +1,17 @@
+STOP=10
+
+start() {
+	generate_mac $WIFI_OTP_REG0
```
generate_mac uses proddata functions so you need to added DEPENDS for proddata too.
However we want to move away from proddata. so can you use wifiaddr from uboot env?
Just setenv wifiaddr for the time being and check if it works?

nikhil-zinjurde-imgtec commented:
retest this please

nikhil-zinjurde-imgtec commented:
retest this please

nikhil-zinjurde-imgtec commented:
retest this please

nikhil-zinjurde-imgtec commented:
retest this please

nikhil-zinjurde-imgtec commented:
retest this please

nikhil-zinjurde-imgtec commented:
retest this please

nikhil-zinjurde-imgtec commented:
retest this please

nikhil-zinjurde-imgtec commented:
retest this please

Ham22 committed:
Merge pull request #94 from CreatorDev/dev

Merge dev to master

Ham22 committed:
Add initial Jenkinsfile

Simply trigger the openwrt job to override this feed and wait for
its results.

Signed-off-by: Ian Pozella <Ian.Pozella@imgtec.com>

Commit:
Remove awalwm2m from Ci40-platform-feed

Signed-off-by: Abhijit Mahajani <Abhijit.Mahajani@imgtec.com>

nikhil-zinjurde-imgtec committed:
Merge pull request #98 from abhijit-mahajani-imgtec/remove_awa

Remove awalwm2m from Ci40-platform-feed

nikhil-zinjurde-imgtec committed:
README.md: update readme with package information

* Remove awalwm2m
* Add tpm-tools and trousers

Signed-off-by: Nikhil Zinjurde <nikhil.zinjurde@imgtec.com>

abhijit-mahajani-imgtec commented:
please update readme too

nikhil-zinjurde-imgtec committed:
Merge pull request #96 from Ham22/jenkinsfile

Add initial Jenkinsfile

Commit:
Merge pull request #99 from nikhil-zinjurde-imgtec/master

README.md: update readme with package information

nikhil-zinjurde-imgtec commented:
@francois-berder-imgtec please add PR directly on master branch, we will soon be closing off the dev branch. So, please close this PR and create a new one on master

abhijit-mahajani-imgtec commented:
Closing this one since its done for master branch in #102
